.github/codecov.yml

```diff
@@ -23,7 +23,32 @@ comment:
   require_changes: no
 
 ignore:
-  - "docs/**/*"
-  - "examples/**/*"
-  - "test/**/*"
   - "**.md"
+  - ".github"
+  - ".vscode"
+  - "build"
+  - "docs"
+  - "examples"
+  - "src/Shared"
```

Comment:
Should shared files under `src` be excluded?

Reply:
Good question. This was copied from contrib but probably shouldn't be here. I'm thinking I'm going to leave it alone to see how codecov handles the new ci and come back to this later to smooth out any rough edges. I haven't figured out a way to make codecov really useful yet on contrib either 🤣